app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/ItineraryCreateActivity.kt

```diff
@@ -94,6 +95,9 @@ class ItineraryCreateActivity : BaseMapActivity(), OnMapReadyCallback,
         const val FLIGHTHEIGHT_INTENT_PATH = "ICA_flightHeight"
         const val DEFAULT_FLIGHTHEIGHT = 50.0
         val DEFAULT_STRATEGY = Strategy.SINGLE_PASS
+
+        // Maximum area size in m2
+        const val MAXIMUM_AREA_SIZE = 100000.0
```

Comment:
A bit oversized maybe 10 000m^2 would be better

Reply:
Yes your right, I'm bad at maths, fortunately we don't get tested on math at EPFL